rust/worker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rust:1.77.2 as builder
+FROM rust:1.79.0 as builder
```

Comment:
needed to bump this, otherwise it was refusing to compile:

<img width="1079" alt="Screenshot 2024-06-28 at 4 25 47 PM" src="https://github.com/chroma-core/chroma/assets/7410405/1a3a5e7b-f6c4-4d14-ba44-8740c0b4b973">

Reply:
totally fine